src/cli/rustup_mode.rs

```diff
@@ -1377,7 +1377,7 @@ fn get_target(m: &ArgMatches, distributable: &DistributableToolchain<'_>) -> Opt
 fn component_remove(cfg: &Cfg, m: &ArgMatches) -> Result<utils::ExitCode> {
     let toolchain = explicit_or_dir_toolchain(cfg, m)?;
     let distributable = DistributableToolchain::try_from(&toolchain)?;
-    let target = get_target(m, &distributable);
+    let target: Option<TargetTriple> = get_target(m, &distributable);
```

Comment:
What requires the type annotation?

Reply:
I guess my RA got some issues. Reverted.